lib/msf/ui/console/module_option_tab_completion.rb

```diff
@@ -213,9 +213,6 @@ def option_values_dispatch(mod, o, str, words)
                 res << port
               end
             end
-            if res.empty?
-              res << rand(1..65534).to_s
-            end
```

Comment:
Can anyone explain why we were returning a random port suggestion when we couldn't find one in the DB?